swim/bootstrap_test.go

```diff
@@ -88,6 +88,22 @@ func (s *BootstrapTestSuite) TestBootstrapDestroy() {
 	s.Error(lerr.Err(), "expected bootstrap to fail if node gets destroyed")
 }
 
+func (s *BootstrapTestSuite) TestJoinHandlerNotMakingAlive() {
+	// get a bootstrapped cluster
+	s.peers = genChannelNodes(s.T(), 3)
+	bootstrapList := bootstrapNodes(s.T(), true, s.peers...)
+
+	s.tnode.node.Bootstrap(&BootstrapOptions{
+		Hosts: bootstrapList,
+	})
+
+	// test that there are no changes to disseminate from the peers after bootstrapping the t(est)node
+	for _, peer := range s.peers {
```

Comment:
Keeping the disseminator assertion is OK with me, but I'd prefer an API like `node.PendingChangeCount()` rather than digging into node -> disseminator -> changes.